styles.css

```diff
@@ -0,0 +1,197 @@
+
+
+body {
+	margin: 0;
+}
+
+.header-main {
+	background-image: url("https://images.unsplash.com/photo-1643330683233-ff2ac89b002c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1664&q=80");
+	background-size: cover;
+	background-repeat:no-repeat;
+	height:800px;
+	display: flex;
+	justify-content: center;
+	flex-direction:column;
+}
+
+h1 {
+	color:white;
+	margin:50px;
+	font-size:60px;
+	font-family: 'Major Mono Display', monospace;
+}
+
+.contact1 {
+	display: flex;
+	background-color: #060606;
+	padding: 10px;
+}
+
+.contact-links{
+	display: flex;
+	margin:10px;
+	font-size:40px;
+	font-family:'Bebas Neue', cursive;
+	color:#FFFFFF;
+	
+}
+
+.contact-links a:hover {
+	transform: scale(1.2);
+	color:#7A0505;
+}
+
+.nav1 {
+	display: flex;
+	background-color: #141B29;
+	padding: 10px;
+}
+
+.nav-links{
+	display: flex;
+	font-size:30px;
+	font-family: 'Major Mono Display', monospace;
+	color:white;
+	transition-duration: .5s ;
+}
+
+.nav-links a:hover {
+	transform: scale(1.2);
+}
+
+.contact {
+	color:white;
+}
+
+.top-links {
+	color:#FFFFFF;
+}
+
+div a {
+	margin-left: 40px;
+}
+
+.header-me {
+	display: flex;
+	flex-direction: column;
+	align-items:center;
+	color:white;
+	
+}
+
+main{
+	background-image: url("https://images.unsplash.com/photo-1605441319085-0a36e0809874?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1160&q=80");
+	background-size: cover;
+	background-repeat:no-repeat;
+	background-position: center center;
+}
+
+.background-img {
+	border-radius:49%;
+	height:auto;
+	width:250px;
+	margin:20px;
+}
+
+.projects {
+		display: flex;
+		flex-wrap:wrap;
+		color:white;
+		margin:60px 60px 0 60px;
+}
+
+.individual-project {
+		display: flex;
+		align-items: center;
+		flex-direction: column;
+		flex:40%;
+		margin:40px;
+		border:outset 3px rgba(0,0,0,0);
+}
+
+.individual-project:hover {
+	border:outset 3px #B5B5B5;
+	border-radius:10%;
+}
+
+.project-img {
```

Comment:
probably could just use a selector like `.individual-project img` to reduce the number of classes floating around the html